pyradiosky/skymodel.py

```diff
@@ -312,7 +312,7 @@ def __init__(
 
         desc = (
             "Component flux per frequency and Stokes parameter. Units compatible with "
-            "one of: ['Jy', 'K str', 'Jy/str', 'K']. Shape: (4, Nfreqs, Ncomponents). "
+            "one of: ['Jy', 'K str', 'Jy/sr', 'K']. Shape: (4, Nfreqs, Ncomponents). "
```

Comment:
Should we change `K str` to be `K sr` then by the same logic or no?

Reply:
Good catch, fixed